test/sql/function/list/list_concat.test

```diff
@@ -61,7 +61,7 @@ foreach type_a <integral> varchar
 foreach type_b <integral> varchar
 
 query T
-SELECT list_concat([1, 2]::${type_a}[], [3, 4]::${type_b}[])
+SELECT list_concat([1, 2]::${type_a}[], [3, 4]::${type_b}[])::INTEGER[]
```

Comment:
Why is this necessary?

Reply:
When integers `type_a` is signed and `type_b` is unsigned, the types in the list get promoted to a larger size signed integer. Considering `HUGEINT` and `UHUGEINT` have no larger type to promote to, they promote to `DOUBLE` which when printed will include the the decimal dot and 0. Casting to a list of integers is purely done to prevent that.

```
SELECT list_concat([1, 2]::hugeint[], [3, 4]::uhugeint[]);
================================================================================
Mismatch on row 1, column 1
[1.0, 2.0, 3.0, 4.0] <> [1, 2, 3, 4]
```

Reply:
Aha, thanks, makes sense